apps/cloud_federation_api/lib/Capabilities.php

```diff
@@ -6,20 +6,27 @@
  * SPDX-FileCopyrightText: 2017 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
-
 namespace OCA\CloudFederationAPI;
 
+use NCU\Security\PublicPrivateKeyPairs\Exceptions\KeyPairException;
```

Comment:
```suggestion
use NCU\Security\KeyPairs\Exceptions\KeyPairException;
```
Might be bikeshedding, but PublicPrivateKeyPair sounds redundant?

Reply:
done